extensions/amp-analytics/0.1/vendors.js

```diff
@@ -63,6 +63,21 @@ export const ANALYTICS_CONFIG = {
           '${baseSuffix}'
     },
     'optout': '_gaUserPrefs.ioo'
+  },
+
+  'snowplow': {
+    'transport': {'beacon': false, 'xhrpost': false, 'image': true},
+    'requests': {
+      'aaVersion': 'amp-0.1',
```

Comment:
I think it can. But I am not sure if others are going to use the same string/format. How about keeping it here for now and moving it to a shared string if others are going to need it?